libs/core/src/lib/tile/directives/tile.directives.ts

```diff
@@ -314,6 +315,14 @@ export class TileActionCloseDirective implements OnInit, OnChanges, CssClassBuil
     elementRef(): ElementRef<any> {
         return this._elementRef;
     }
+
+    /** @hidden */
+    private _addCloseIcon(): void {
+        const element = document.createElement('i');
```

Comment:
@JKMarkowski What if we create a function that will be generated an icon element and will be used in two places?
something like this
```typescript
private _addCloseIcon(): void {
    const element = generateIcon('decline');
    this._elementRef.nativeElement.appendChild(element);
}

function generateIcon(iconName: string): HTMLElement {
    const element = document.createElement('i');
    element.setAttribute('role', 'presentation');
    element.classList.add(`sap-icon--${iconName}`);
    
    return element;
}
```